instrumentation/spring/spring-cloud-aws-3.0/javaagent/build.gradle.kts

```diff
@@ -22,7 +22,7 @@ dependencies {
 
   testImplementation("org.elasticmq:elasticmq-rest-sqs_2.13")
 
-  testImplementation("io.awspring.cloud:spring-cloud-aws-starter-sqs:3.0.0")
```

Comment:
line 16 indicates that we should be using `testImplementation`, should we update that comment?